src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs

```diff
@@ -84,31 +82,15 @@ public void StartMigrationWizard(BoundSonarQubeProject oldBinding)
             }
         }
 
-        private Action GetTfVcHelpAction()
+        private bool IsUnderGit
```

Comment:
property is only used once

Reply:
yes, It looks cleaner on the method.

Reply:
It is confusing tho. First of all, it's just on ==. Secondly, semanticaly this is a method, not a property. If it's a property I assume that it's part of the data stored by object, but you're using it instead of local variable/method.